tools/pylint-extensions/azure-pylint-guidelines-checker/README.md

```diff
@@ -1,103 +1,77 @@
 # Linting the Guidelines
 
-The code presented in this repository is a lift-and-shift of code [originally written by @kristapratico](https://github.com/Azure/azure-sdk-for-python/blob/ae2948416956938c908dcfb570a7456b23482149/scripts/pylint_custom_plugin/)
```

Comment:
Oh I see how it is ;)

Reply:
I swear it went into contribution.md

Reply:
oh oop and then got deleted but that was the intent